src/screens/dashboard/dashboard.tsx

```diff
@@ -452,6 +453,7 @@ export class DashboardScreenComponent extends React.Component<
 
         return (
             <View testID="dashboard-screen" style={[styles.container, { height: containerHeight }]}>
+                <StakingBetaBadge />
```

Comment:
we should add a throttle for this, so we will able to display and hide without a app release. Talk with George about this, it's very simple :)